tests/Platforms/SqlitePlatformTest.php

```diff
@@ -824,4 +824,9 @@ public function testGetCreateTableSQLWithColumnCollation() : void
             'Column "no_collation" will use the default collation (BINARY) and "column_collation" overwrites the collation on this column'
         );
     }
+
+    public function testGetSequencePrefixWithSchema() : void
+    {
+        self::assertEquals('bar__foo', $this->platform->getSequencePrefix('foo', 'bar'));
```

Comment:
Here should be 
```suggestion
        self::assertSame('bar__foo', $this->platform->getSequencePrefix('foo', 'bar'));
```

Maybe we can propose this stricter check with `phpstan/phpstan-unit`.